docs/UserGuide.md

```diff
@@ -165,44 +165,98 @@ Format: `list`
 
 [Back to Table of Contents](#table-of-contents)
 
-#### Locating persons by name: `find`
+#### Locating patients by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds patients whose names contain any of the given keywords.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
+{: .alert .alert-info} 
+> **:information_source: **Note:**
+> 
+> * The search is case-insensitive. e.g `hans` will match `Hans`
+> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+> * Only the name is searched.
+> * Only full words will be matched e.g. `Han` will not match `Hans`
+> * Patients matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+> * Returns an empty patient list panel if no matching patients with the given keywords are found.
+>
+> Examples:
+> * `find John` returns `john` and `John Doe`
+> * `find alex david` returns `Alex Yeoh`, `David Li`
+>  ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+[Back to Table of Contents](#table-of-contents)
+
+#### Locating patients by NRIC: `findMedCon`
```

Comment:
Would need to change this to Locating patients by their medical conditions

Reply:
Resolved